content/400-reference/200-api-reference/100-prisma-schema-reference.mdx

```diff
@@ -188,11 +188,12 @@ The following tables list all supported operating systems with the name of platf
 | :------- | :----------------------- |
 | Windows  | `windows`                |
 
-##### Linux (Alpine)
+##### Linux (Alpine) x86_64
 
-| Build OS | Prisma engine build name |
-| :------- | :----------------------- |
-| Alpine   | `linux-musl`             |
+| Build OS             | Prisma engine build name   | OpenSSL |
+| :------------------- | :------------------------- | :-----: |
+| Alpine (3.17+)       | `linux-musl-openssl-3.0.x` |  3.0.x  |
```

Comment:
While @aqrln and I initially though about this as well, this would be a novel approach to the document, as we haven't ever mentioned the Prisma version an engine started appearing in in the past.

Reply:
Indeed, maybe something for the docs team to think about

Reply:
We should totally document that for things that were not available earlier.

At the same time, these should also be documented somewhere else and not just in this list here which would be a really difficult place to put all this information.

Reply:
I think documenting on here with an asterisk would work for now, at least we get the information in somehow. Agree we need to think more about how to do this in general.

Reply:
(just noticed that it explains exactly what I asked in the issue description)

Reply:
I've added the asterisk to the table

Reply:
Also made #4287 to track documenting OpenSSL etc properly somewhere